bin/xud

```diff
@@ -27,11 +27,20 @@ const { argv } = require('yargs')
       describe: 'Format of the logger date',
       type: 'string',
     },
-    network: {
-      describe: 'The network the lnd clients are using',
-      type: 'string',
-      choices: ['mainnet', 'testnet', 'simnet', 'regtest'],
-      alias: 'n',
+    mainnet: {
+      describe: 'Whether to run XUD on mainnet',
+      type: 'boolean',
+      default: undefined,
+    },
+    testnet: {
+      describe: 'Whether to run XUD on testnet',
+      type: 'boolean',
+      default: undefined,
+    },
+    regnet: {
```

Comment:
Having the default value there is problematic, because since we want to get only one "alternative" network, it will need to be explicitly disabled. Unless you want me to add `simnet` there without specifying it as the default (which I found to be confusing).

Reply:
I was thinking we can add the `--simnet` option here and still set the `default` as `undefined` for the command line args at least. That's consistent with how we're treating other flags that are enabled by default if unspecified in the command line.